config/locales/views/pages/en.yml

```
@@ -34,17 +34,52 @@ en:
       no-activities-help: "You have solved all exercises!"
       start-activity: "Start with %{activity}"
       continue-activity: "Continue working on %{activity}"
+      unanswered-questions:
+        one: "One open question"
+        other: "%{count} open questions"
+      unanswered-questions-subtitle:
+        one: "Click here to view this question"
+        other: "Click here to view these questions"
+      pending-members:
+        one: "One pending member"
+        other: "%{count} pending members"
+      pending-members-subtitle:
+        one: "Click here to view this member"
+        other: "View all users that want to join this course"
+      incomplete-feedbacks:
+        one: "One unreviewed submission"
```

In the evaluations UI, we use the term "complete", so perhaps something like this:

```suggestion
        one: "One incomplete submission"
```

---

this would also require a change to the word submission. (As we are not completing the submission, but completing the process of reviewing the submission)

---

I don't really agree here. In the evaluations UI, you already have the evaluation context which is why "complete" is good enough there. On the home page, you don't have that context and "unreviewed" provides that.